CGF.py

```diff
@@ -0,0 +1,13 @@
+
+#given a list find the greatest commmon factor,( the largest number that is divisible by all 
+# items in the list
+
+def cdf(arr):
```

Comment:
why is this function called "cdf"? if it's calculating greatest common divisor, this abbreviation doesn't make sense to me... am I missing something?